src/Symfony/Component/Validator/Mapping/Cache/PsrCache.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Mapping\Cache;
+
+use Psr\Cache\CacheItemPoolInterface;
+use Symfony\Component\Validator\Mapping\ClassMetadata;
+
+/**
+ * Adapts a PSR-6 cache to a CacheInterface.
+ *
+ * @author David Maicher <mail@dmaicher.de>
+ */
+class PsrCache implements CacheInterface
```

Comment:
Wanted to name it `PsrCacheAdapter` but as the existing doctrine adapter is named `DoctrineCache` I followed the pattern.